fe/fe-core/src/main/java/com/starrocks/common/Log4jConfig.java

```diff
@@ -309,7 +309,8 @@ private static String generateXmlConfTemplate() {
         }
 
         String newXmlConfTemplate = APPENDER_TEMPLATE;
-        newXmlConfTemplate += Config.sys_log_to_console ? CONSOLE_LOGGER_TEMPLATE : FILE_LOGGER_TEMPLATE;
+        newXmlConfTemplate += (Config.sys_log_to_console || System.getenv("SYS_LOG_TO_CONSOLE").trim().equals("1"))
```

Comment:
I mean, if we can make the change in Config.java, something like

```
public static boolean sys_log_to_console = System.getenv("SYS_LOG_TO_CONSOLE").trim().equals("1");
```

Reply:
updated